articles/best-practices-for-rich-text.mdx

```diff
@@ -3,60 +3,43 @@ _schema: default
+* You can unintentionally delete custom markup from your files. As custom markup cannot be recreated in the rich text editor (i.e., it is invisible), it is easy to accidentally backspace or delete your HTML structures.
+* Some custom markup structure pose a risk for prevent cross-site scripting attacks (i.e., `<script>` tags) and other unwanted interactions between your code and the editing interface (i.e., `<style>` tags).
 
-<comp.DocsImage path="/documentation/static/boxed-xss-elements.png" alt="script and style elements replaced with boxes" type="screenshot-snippet"/>
+Although you cannot edit your custom markup in the Content Editor or rich text inputs, CloudCannon will preserve it. Your custom markup will appear as an uneditable Snippet.
 
-Some elements with simple text content, like `kbd` and `main`, will be included in this process and will become uneditable. These elements always also fall into the category of HTML that cannot be reconstructed, and should be avoided in rich text content anyway.
+<comp.DocsImage path="https://cc-dam.imgix.net/documentation/images/editing/unsupported-html-rich-text/2024-Q1/CloudCannon-Documentation-Unsupported-HTML-Snippets.png" alt="Unsupported HTML in the rich text editor will appear as an uneditable Snippet." title="Uneditable Snippets in rich text" type="ui-snippet"/>
 
-The full list of elements that will be replaced with boxes is: `base`, `blink`, `button`, `canvas`, `data`, `datalist`, `dfn`, `dialog`, `fieldset`, `form`, `hgroup`, `iframe`, `input`, `kbd`, `label`, `legend`, `link`, `main`, `map`, `marquee`, `math`, `menu`, `menuitem`, `meta`, `meter`, `noscript`, `object`, `optgroup`, `option`, `output`, `picture`, `portal`, `progress`, `q`, `rb`, `rp`, `rt`, `ruby`, `samp`, `script`, `select`, `slot`, `source`, `style`, `svg`, `template`, `textarea`, `time`, `title`, `track`, `var`, and `wbr`.
+You can still edit the surrounding content in your files using a rich text editor. You can also edit your custom markup in the Source Editor.
 
-Any content that is not boxed this way will also be processed with a sanitization library, to remove potential vulnerabilities or clashes with the CloudCannon interface. This will remove most attributes from HTML elements. The only exceptions are the `class` attribute, and other basic attributes required for the functionality of different elements (e.g. `src` and `alt` for `img`). Once again, these attributes also fall into the category of HTML that cannot be reconstructed, and should be avoided in rich text content anyway.
+### Best practice
 
-## Best practices
+To prevent your files from becoming uneditable in the rich text editor, we recommend preparing your rich text files appropriately.
```

Comment:
```suggestion
To ensure your files remain editable in the rich text editor, we recommend the solutions below to avoid some common pitfalls:
```

Maybe not this exactly, but it might be nice to have a more positive perspective